src/components/Footer.tsx

```diff
@@ -98,7 +98,7 @@ const Footer = () => {
             </li>
             <li className="mb-4">
               <a
-                href="https://moloch.daohaus.fun"
+                href="https://moloch.daohaus.club"
```

Comment:
This one doesn't seem like it should be .club

this is an old pokemol deployment:
https://moloch.daohaus.club/

Probably want:
https://moloch.daohaus.fun/